docs/source/install.rst

```diff
@@ -35,7 +35,7 @@ environment.
     :linenos:
 
     export INSTALL_ENVIRONMENT_NAME=choose_your_own_name
-    conda create -y -n "${INSTALL_ENVIRONMENT_NAME}" python pyqt=5
+    conda create -y -n "${INSTALL_ENVIRONMENT_NAME}" python pyqt=5 pyepics
```

Comment:
add python=3.10

Reply:
Not necessary to pin to _only_ 3.10.  Let the solver and the package requirements decide that.

Reply:
ok

Reply:
For example, this combination works on my APS workstation:

```bash
$ conda list python
# packages in environment at /home/beams/JEMIAN/.conda/envs/model_instrument:
#
# Name                    Version                   Build  Channel
dnspython                 2.6.1                    pypi_0    pypi
ipython                   8.28.0                   pypi_0    pypi
python                    3.12.6          hc5c86c4_2_cpython    conda-forge
python-dateutil           2.9.0.post0              pypi_0    pypi
python-json-logger        2.0.7                    pypi_0    pypi
python-multipart          0.0.12                   pypi_0    pypi
python_abi                3.12                    5_cp312    conda-forge
types-python-dateutil     2.9.0.20241003           pypi_0    pypi
```